README.md

```diff
@@ -8,7 +8,8 @@ This Project is created purposely to learn how to build Portfolio.
 
 ## Requirements
 * Create a new GitHub Repo and a new branch with GitHub flow
-*  Setup Linter for HTML and CSS
+* Setup Linter for HTML and CSS
+* Add Form validation 
 
 ## Getting Started
```

Comment:
- [ ] Kindly recheck your readme file, I am afraid the information in your getting started section is not the required one, the getting started section is supposed to help visitors to your repo know how to cone and run your project locally, what they will need and the steps to follow.